app/views/works/_work_blurb.html.erb

```diff
@@ -1,17 +1,21 @@
 <% # expects "work" %>
-<li id="work_<%= work.id %>" class="<% if is_author_of?(work) %>own <% end %>work <%= css_classes_for_creation_blurb(work) %>" role="article">
-  <%= render 'works/work_module', :work => work %>
+<li id="work_<%= work.id %>" class="<% if is_author_of?(work) %>own <% end %><% if work.unrevealed? %>mystery <% end %>work <%= css_classes_for_creation_blurb(work) %>" role="article">
```

Comment:
Is this change just for consistency with the gift blurb?

Reply:
Yup! (And on the subject of consistency, I used `mystery` rather than `unrevealed` because that's what the work module uses in [the header module](https://github.com/otwcode/otwarchive/blob/808b34b1d21be65c26fc9ae4a20621c40955a37d/app/views/works/_mystery_blurb.html.erb#L1).)